printer/printSubpod.js

```diff
@@ -21,7 +21,10 @@ function printPlaintext(output, plaintext, options) {
 
   // Print multiple lines of plaintext
   if (plaintext[0].length !== plaintext[1].length) {
-    output.putText(plaintext.map(line => line.join('')).join('\n'));
+    output.putText(plaintext.filter(line => line.join(''))
+                            .map(line => line.filter(line => line).join(' '))
+                            .join('\n')
+                  );
```

Comment:
The first filter filters out a plaintext array which consists only of empty strings.
The second one filters out empty strings from each non-empty plaintext array.

@vladimyr 
This solution is primitive. Maybe try parsing out empty values beforehand in https://github.com/vladimyr/wolfram-query?